packages/app-project/next.config.js

```diff
@@ -25,7 +25,7 @@ const PANOPTES_ENV = process.env.PANOPTES_ENV || 'staging'
 const webpackConfig = require('./webpack.config')
 const assetPrefix = process.env.PROJECT_ASSET_PREFIX || basePath
 const SENTRY_PROJECT_DSN = process.env.SENTRY_PROJECT_DSN
-const APP_ENV = process.env.APP_ENV || 'production'
+const APP_ENV = process.env.APP_ENV || 'development'
```

Comment:
This line exists in app-content-pages too, should both NextJS apps have the same config?

Reply:
Probably. Otherwise local development errors could be sent to Sentry as production errors.